Python/Product/PythonTools/PythonTools/Commands/SendToReplCommand.cs

```diff
@@ -52,6 +52,10 @@ class SendToReplCommand : Command {
             _serviceProvider = serviceProvider;
         }
 
+        private static bool IsCellMarker(ITextSnapshotLine line) {
+            return line.GetText().Trim(' ', '\t').StartsWith("#%%");
```

Comment:
Might have been nicer if it used the same code as the tagger to determine if it's a cell marker, but I guess it's unlikely to be something that causes maintenance issues.